src/main/kotlin/com/faforever/userservice/ui/view/ucp/UcpChangeUsernameView.kt

```
@@ -0,0 +1,112 @@
+            },
+        )
+
+        add(submitButton)
+    }
+
+    private fun handleChangeUsername() {
+        val currentUser = ucpSessionService.getCurrentUser()
+        val newUsername = newUsernameField.value
+        val result = try {
+            ucpUsernameService.changeUsername(currentUser, newUsername)
+        } catch (exception: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)